backend/app/controllers/white.list.domain.controller.js

```diff
@@ -0,0 +1,24 @@
+class WhiteListDomainController {
+  constructor(whiteListDomains) {
+    if(whiteListDomains!=""){
+    	this.whitelistDomains = JSON.parse(whiteListDomains);
+    }else{
+    	this.whitelistDomains = "";
```

Comment:
```suggestion
    } else {
    	this.whitelistDomains = [];
```

As there is a default, do we really need the `else` here?